csm/core/agent/csm_agent.py

```diff
@@ -245,10 +245,11 @@ def run():
     from cortx.utils.log import Log
     from csm.common.runtime import Options
     Options.parse(sys.argv)
+    if not Options.start:
+        os._exit(1)
```

Comment:
Before exiting please add a print statement specifying the user to add start argument. A kind of usage to start csm agent.

Reply:
Comment resolved